lib/kaffe/group_member/manager/group_member_supervisor.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Kaffe.GroupMemberSupervisor do
+  @moduledoc """
+  The top-level supervisor for group members and subscribers.
+  """
+
+  use Supervisor
+
+  require Logger
+
+  def start_link do
+    Supervisor.start_link(__MODULE__, :ok)
+  end
+
+  def start_worker_supervisor(supervisor_pid) do
+    Supervisor.start_child(supervisor_pid, supervisor(Kaffe.WorkerSupervisor, []))
```

Comment:
It's not really passing along to another supervisor but telling the supervisor to provision a child, which happens to be a supervisor.